extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -62,10 +69,11 @@ public function executeShell() {
 		global $IP;
 
 		$cmd = sprintf(
-			"SERVER_ID=%d %s %s/maintenance/importStarter.php",
+			"SERVER_ID=%d %s %s/maintenance/importStarter.php --vertical=%d",
 			$this->taskContext->getCityId(),
 			$this->phpBin,
-			"{$IP}/extensions/wikia/CreateNewWiki"
+			"{$IP}/extensions/wikia/CreateNewWiki",
+			$this->taskContext->getVertical()
```

Comment:
Can't we get vertical from WikiFactory? `importStarter.php` maintenance script is run in the context of newly created wiki that is set up partially at that point.